summoner-cli/src/Summoner/GhcVer.hs

```diff
@@ -28,6 +29,13 @@ data GhcVer
     | Ghc865
     deriving (Eq, Ord, Show, Enum, Bounded)
 
+showGhcOutput :: GhcVer -> [Text]
+showGhcOutput ghcVer = 
+    [showGhcVer ghcVer
+    ,"base-" <> baseVer ghcVer
+    ," lts-" <> latestLts ghcVer
+    ]
```

Comment:
Let's use a tuple and write this code following our style guide like this (and changing kinda generic name `showGhcOutput` to `showGhcMeta`)

```haskell
-- | This functions shows GHC version
showGhcMeta :: GhcVer -> (Text, Text, Text)
showGhcMeta ghcVer =
    ( "ghc-"  <> showGhcVer ghcVer
    , "base-" <> baseVer ghcVer
    , "lts-"  <> latestLts ghcVer
    )
```